.github/workflows/build-docker-corsika-simtelarray-image.yml

```diff
@@ -205,3 +210,52 @@ jobs:
           file: ./docker/Dockerfile-prod-opt
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}-prod-opt
+      - name: set image tag
+        run: |
+            echo "image_tag=$(echo '${{ steps.meta.outputs.tags }}' | head -n1)" >> "$GITHUB_ENV"
+        id: set-image-tag
+    outputs:
+      image_tag: ${{ steps.meta.outputs.tags }}
+
+  test-images:
+    runs-on: ubuntu-latest
+    needs: build-production-images-optimized
+    strategy:
+      matrix:
+        version:
+          - {simtel_version: '240205', corsika_version: '77500', bernlohr_version: '1.67'}
+          # - {simtel_version: '240927', corsika_version: '77550', bernlohr_version: '1.68'}
+        hadronic_model: ['qgs2']
+        production:
+          - {build_opt: 'prod6-sc', extra_def: '-DMAXIMUM_TELESCOPES=128'}
+          - {build_opt: 'prod6-baseline', extra_def: ''}
+        avx_instruction: ['no_opt']
+    container:
+      image: ghcr.io/gammasim/simtools-prod-opt-${{ matrix.version.simtel_version }}-corsika-${{ matrix.version.corsika_version }}-bernlohr-${{ matrix.version.bernlohr_version }}-${{ matrix.production.build_opt }}-${{ matrix.hadronic_model }}-${{ matrix.avx_instruction }}
```

Comment:
What's the `opt` in `prod-opt` here? 

Reading further, I understand it comes from Docker image name. Are they still separate and the `opt` is needed?

Reply:
Removed the `opt`. Good point.